src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -1236,4 +1506,110 @@ internal class OnboardingSupervisor(
             helper.send(error, callback)
         }
     }
+
+    @Suppress("ForbiddenComment")
+    private fun cctpToNobleSkip(
+        state: PerpetualState?,
+        decimals: Int,
+        gas: BigDecimal,
+        accountAddress: String,
+        subaccountNumber: Int?,
+        callback: TransactionCallback
+    ) {
+//      We have a lot of duplicate code for these deposit/withdrawal route calls
+//      It's easier to dedupe now that the url is the same and only the args differ
+//      TODO: Consider creating generateArgs fun to reduce code duplication
```

Comment:
nit please make tickets instead of leaving TODOs

Reply:
if this is something you are going to tackle as a follow up, can just leave a comment in the pr